pydy/codegen/ode_function_generators.py

```diff
@@ -169,8 +169,8 @@ def _deduce_system_type(**kwargs):
         full mass matrix
             M(x, p) * x' = f(x, t, r, p)
         min mass matrix
-            M(q, p) * u' = f(q, u, t, r, p)
-            q' = g(q, u, t)
+            M(x, p) * x'_d = f(x, t, r, p)
+            x'_k = g(x, t)
```

Comment:
I was trying to indicate the difference between the a "dynamic" states x'_d and "kinematic" states x'_k, which was me trying to generalize out the generalized coordinates and speeds. I'm happy to revert back, it just seemed a bit odd to keep ``q`` and ``u`` since they aren't actually kept separately. But I have no strong opinion on the notation.

I do want to to keep the minimum mass matrix form usable, however I don't recall the use-case for it and I couldn't see how that form could be used based on the current implementation. If there is an example using this form, I can make sure it works.